agent_identity/src/document/aggregate.rs

```diff
@@ -534,23 +534,30 @@ pub mod document_tests {
         document_id: String,
         did_method: SupportedDidMethod,
         document: CoreDocument,
-        document_with_verification_method: CoreDocument,
+        document_with_es256_verification_method: CoreDocument,
+        document_with_both_verification_methods: CoreDocument,
```

Comment:
I suggest renaming this to `multiple_verification_methods` (and possible the one that only contains es256 to `single_verification_method`).